Clinical-Genomics/scout
# add info on HGNC primary transcript to general report

Show primary HGNC reference transcript in general report. fix #1580 

**How to test**:
1. On a local instance of scout, master branch, pin a variant (POT1 is perfect for testing the PR)
1. Note that on general report any info regarding primary HGNC transcript is missing.
1. Switch to this branch and reload the report. The above info should now be present. 

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
